_syllabus/seed.py

```diff
@@ -76,12 +77,19 @@
         hwDate  = lecDate.replace(hours=15)
         solDate = hwDate.shift(weeks=1)
 
+
+	 
+	oh_plus_plus_link = wk{0:02d}-ohpp.md'.format(idx + 1)
+
```

Comment:
I added that line to potentially be used at the beginning of the semester to pre-populate it. However, when making the current changes, adding the links manually was easier since it didn't mess up the previously entered data. So my change to seed.py may not be useful